taskcluster/ac_taskgraph/__init__.py

```diff
@@ -58,6 +58,6 @@ def get_decision_parameters(graph_config, parameters):
         if head_tag[1:] != version:
             raise ValueError(
                 "Cannot run github-release if tag {} is different than in-tree "
-                "{version} from buildconfig.yml".format(head_tag[1:], version)
```

Comment:
Oups, mea culpa. Thanks for fixing this! I had f-strings in mind last time

Reply:
No problem! I didn't see the first time either. I wish we migrated taskgraph to py3 so everything can be f-strings! 🙂